src/client/Microsoft.Identity.Client/MsalException.cs

```diff
@@ -42,6 +42,11 @@ public class MsalException : Exception
         /// </summary>
         public const string BrokerErrorCode = "BrokerErrorCode";
 
+        /// <summary>
+        /// An <see cref="AdditionalExceptionData"/> property key, available when using desktop brokers.
+        /// </summary>
+        public const string BrokerTelemetry = "BrokerTelemetry";
```

Comment:
>BrokerTelemetry [](http://example.com/codeflow?start=47&length=15)

If we select a more generic name "TelemetryBlob" or something like this we can re-used it even for non-brokered flows....